### PR TITLE
Add myself as a participant

### PR DESCRIPTION
**Pull request by makmic**
I would like to register for JS CraftCamp.

- [x] My pull request contains a JSON file `$name_or_nickname.json`
- [x] I read the `THE IMPORTANT STUFF` at [https://jscraftcamp.org/registration](https://jscraftcamp.org/registration), the JSON file follows the [template](https://github.com/jscraftcamp/website/blob/main/participants/_template.json), and contains the mandatory fields like `realName`, `githubAccountName`, `when`, `iCanTakeNotesDuringSessions`, `whatIsMyConnectionToJavascript` and `whatCanIContribute`.
- [x] If I can NOT attend, I will either send another pull request removing my JSON file or an e-mail with the subject 'UNREGISTER' to team@jscraftcamp.org
- [x] I agree that the data I enter in the registration can be used for running the event, e.g. make a name tag, count me as participant. I acknowledge that my data are public on GitHub and I agree that I will be listed on the JSCraftCamp website as a participant.
- [x] I agree that photos and videos might be taken and published (e.g. on social media) during the event.
- [x] I understand that I might NOT get a T-Shirt, because they might be in production already

### CODE REVIEW

File: participants/celestine_auburger.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
 		"placeFamilyNameFirst": false,
 		"hideFamilyNameOnWebsite": false
 	},
-	"githubAccountName": "irenama",
+	"githubAccountName": "aubcel",
```
[the remark this answers is not in the document]
Comment (Narigo) [reply participants/celestine_auburger.json:8]: Thanks for mentioning this! We have to wait with merging this PR though as we might need to put @makmic on the waiting list due to some sponsors still needing to register and we can't really overbook too much.

[the remark this answers is not in the document]
Comment (aubcel) [reply participants/celestine_auburger.json:8]: Oh no! What a pity 🙁 
🤞🏻 that he can come on saturday :)

**Comment by makmic** [comment]
> Hi @makmic - as you have probably seen, we have to put you on the waiting list for now. One question though, if there are people unregistering again: Is Saturday only attendance intentional?
> 
> I'd love to hear something about the Web Audio API and now that I googled it, I wonder how Unpoly is similar or even better than HTMX 🤔

Hi @Narigo,
yes I've noticed, thanks for the heads up though! Saturday only is intentional as I cannot make it on Friday unfortunately.

Regarding htmx vs. Unpoly: I'd say they solve similar problems with slightly different philosophies. Both are good on what they want to archive :)

**Comment by Narigo** [comment]
Welcome to the JSCC24 @makmic ! Looking forward to seeing you there! 🥳

**Pull request merged**